### PR TITLE
Updated GraphML demo

### PR DESCRIPTION
**Pull request by d-michail**
Updated GraphML demo to show both importing and exporting from GraphML.

### CODE REVIEW

File: jgrapht-demo/src/main/java/org/jgrapht/demo/GraphMLDemo.java [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+            System.out.println(graph1AsGraphML);
+
+            // import it back
+            System.out.println("-- Importing graph back from GraphML");
+            Graph<CustomVertex, DefaultWeightedEdge> graph2 =
+                new DirectedWeightedPseudograph<>(DefaultWeightedEdge.class);
+            GraphImporter<CustomVertex, DefaultWeightedEdge> importer = createImporter();
+            importer.importGraph(graph2, new StringReader(graph1AsGraphML));
+            
+            // export again as string
```
Comment (jkinable) [inline jgrapht-demo/src/main/java/org/jgrapht/demo/GraphMLDemo.java]: What is the purpose of exporting it again, ie.. what is the learning value here?

**Comment by jkinable** [comment]
Looks good. Added one question.

File: jgrapht-demo/src/main/java/org/jgrapht/demo/GraphMLDemo.java (hunk 1 above)
Comment (d-michail) [inline jgrapht-demo/src/main/java/org/jgrapht/demo/GraphMLDemo.java]: You are probably right, I just wanted to show that the output is the same. I will change it.

**Comment by jkinable** [comment]
Thanks! Always good to have more demo's

**Pull request merged**